packages/gateway/src/Shard.ts

```diff
@@ -568,10 +582,11 @@ export class DiscordenoShard {
           this.logger.debug(`[Shard] Heartbeat not acknowledged for shard #${this.id}. Assuming zombied connection.`)
           this.close(ShardSocketCloseCodes.ZombiedConnection, 'Zombied connection, did not receive an heartbeat ACK in time.')
 
-          return await this.identify()
+          await this.resume()
```

Comment:
Is this typo? Why was it changed from identify to resume?

Reply:
> If a client does not receive a heartbeat ACK between its attempts at sending heartbeats, this may be due to a failed or "zombied" connection. The client should immediately terminate the connection with any close code besides 1000 or 1001, then reconnect and attempt to Resume.

From the discord docs, https://discord.com/developers/docs/topics/gateway#heartbeat-interval-example-heartbeat-ack